app/tests/src/common/pool-reg-tx-settings.js

```diff
@@ -0,0 +1,176 @@
+export const poolRegTxSettings = {
```

Comment:
poolRegTxFixtures would imho look more natural as a name. "Settings" evokes some generic configuration transcending a single pool registration tx, which isn't the case here

Reply:
but I see at the same time the "settings" suffix is widely used, and it's probably not worth it deviating/changing the previous instances if it is already a coined term